osu.Game.Rulesets.Osu/UI/OsuResumeOverlay.cs

```diff
@@ -172,13 +173,14 @@ public OsuResumeOverlayInputBlocker()
                 Depth = float.MinValue;
             }
 
-            public bool OnPressed(KeyBindingPressEvent<OsuAction> e)
+            protected override void Update()
             {
-                bool block = BlockNextPress;
+                base.Update();
                 BlockNextPress = false;
```

Comment:
I feel like this needs local documentation? On its own it looks like a bug.

Reply:
I tried adding explanatory note on this one but I realised how hacky it makes the whole fix sound, so I ended up rewriting the fix to make more sense.